lxd/db/containers.go

```diff
@@ -914,6 +911,15 @@ SELECT containers.name
 	return max
 }
 
+// ContainerNextSnapshot returns the index the next snapshot of the container
+// with the given name should have.
+//
+// Note, the code below doesn't deal with snapshots of snapshots.
+// To do that, we'll need to weed out based on # slashes in names
+func (c *Cluster) ContainerNextSnapshot(project string, name string) int {
```

Comment:
Can we drop that one? it seems to have a single user, so we should be able to just move that to a call to ContainerNextSnapshotWithPrefix with "snap" as prefix.